Missing GoogleTest sources do not disable test projects

I started to setup a Linux machine to compile PCL and missed to install GoogleTest.

Expected: BUILD_global_tests is automatically disabled, because of missing GoogleTest dependency
Current:
```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
GTEST_INCLUDE_DIR (ADVANCED)
   used as include directory in directory /media/sf_pcl/test
...
GTEST_SRC_DIR (ADVANCED)
   used as include directory in directory /media/sf_pcl/test
...
Configuring incomplete, errors occurred!
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -421,12 +421,6 @@ configure_file("${pcl_config_h_in}" "${pcl_config_h}")
 PCL_ADD_INCLUDES(common "" "${pcl_config_h}")
 include_directories("${CMAKE_CURRENT_BINARY_DIR}/include")
 
-### ---[ Set up for tests
-enable_testing()
-
-### ---[ Set up for examples
-#include("${PCL_SOURCE_DIR}/cmake/pcl_examples.cmake")
-
 ### ---[ Add the libraries subdirectories
 include("${PCL_SOURCE_DIR}/cmake/pcl_targets.cmake")
 

--- a/cmake/Modules/FindGTestSource.cmake
+++ b/cmake/Modules/FindGTestSource.cmake
@@ -32,7 +32,8 @@ set(GTEST_INCLUDE_DIRS ${GTEST_INCLUDE_DIR})
 set(CMAKE_FIND_FRAMEWORK)
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(Gtest DEFAULT_MSG GTEST_INCLUDE_DIR GTEST_SRC_DIR)
+find_package_handle_standard_args(GTestSource DEFAULT_MSG GTEST_INCLUDE_DIR GTEST_SRC_DIR)
+set(GTEST_FOUND ${GTestSource_FOUND})
 
 mark_as_advanced(GTEST_INCLUDE_DIR GTEST_SRC_DIR)
 

